conf/aws.yaml.template

```
@@ -31,6 +31,9 @@ AWS:
             # Specified as {time_value}{time_unit} format, e.g. "7d" = 7 Days
             # If a time unit is not specified (the value is numeric), it will be considered as Minutes
             SLA: 7d
+            # Specify OCP clusters prefix to filter by, for example: '["my_project_", "openshift-"]' etc..
+            # Don't use '*' to express the use of a prefix
+            OCP_PREFIX: '[]'
```

```suggestion
            OCP_PREFIX: []
```
I would also rename it to OCP_PREFIXES.

---

It doesn't work with dynaconf. It's required to use `'[]'` through env vars. Verified.

---

Yeah, through env var its ok, but since we are giving that in YAML, it should be given a list. See similar examples in the YAML file itself.